src/build_platform/mod.rs

```diff
@@ -94,6 +94,9 @@ impl Image {
         self.registry_url.host_str().unwrap()
     }
 
+    pub fn repository_name(&self) -> &str {
+        self.name.split('/').collect::<Vec<&str>>()[0]
```

Comment:
What about computing this field only once at Image creation?

Reply:
done